lib/Elastica/Index.php

```diff
@@ -420,11 +441,10 @@ public function addAlias($name, $replace = false)
      */
     public function removeAlias($name)
     {
-        $path = '_aliases';
+        $endpoint = new \Elasticsearch\Endpoints\Indices\Alias\Delete();
+        $endpoint->setName($name);
```

Comment:
No, it is not needed, since we call $this->requestEndpoint(...) it will add index name there.